distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1175,6 +1175,26 @@ type Column
         new_name = self.naming_helper.function_name "ends_with" [self, other]
         make_text_case_op self "ends_with" other case_sensitivity new_name
 
+    ## GROUP Standard.Base.Text
+       ICON preparation
+       Calulates the text length for each element of the column
+
+       In the Database backends, the default text length method of the
+       particular database is used.
+
+       In the in-memory backend, this will give you the grapheme length of the string.
+
+       > Example
+             import Standard.Examples
+
+             example_text_length =
+                Examples.text_column_1.text_length
+    text_length : Column
+    text_length self =
+        Value_Type.expect_text self <|
+          new_name = self.naming_helper.function_name "text_length" [self]
+          self.make_unary_op "length" new_name
```

Comment:
We usually try to keep the indentation of 4 spaces for nested blocks.

Although I think it may indeed be worth reconsidering that rule, to make bigger blocks more compact.

Reply:
Fixed